### DISCUSSION TITLE
Storybook and Next/Image: Impossible to have an Image without width and height in Storybook

### DISCUSSION THREAD

**Discussion by WebPierre** [vercel/next.js#33010] [Help]
### What version of Next.js are you using?

12.0.7

### What version of Node.js are you using?

16.13.1

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

When I add a local image with the next/image component without specify width and height according to [https://nextjs.org/docs/basic-features/image-optimization#local-images](url) I always have the error:
`Image with src "static/media/public/images/x.png" must use "width" and "height" properties or "layout='fill'" property.` on Storybook.

I don't have any problem outside Storybook.

### Expected Behavior

I would like not to add, just for Storybook, width and height for local image.

### To Reproduce

- Use the example `with-storybook` [https://github.com/vercel/next.js/tree/canary/examples/with-storybook](url)
- In `public` directory, add an image 
- In `index.js`, import this image and use the `next/image` component with only `src`, according to [https://nextjs.org/docs/basic-features/image-optimization#local-images](url)
- In `.storybook/main.js`, add `staticDirs: ["../public"]`
- In `.storybook/preview.js`, add
```
import * as NextImage from "next/image";

const OriginalNextImage = NextImage.default;

Object.defineProperty(NextImage, "default", {
  configurable: true,
  value: (props) => <OriginalNextImage {...props} unoptimized />,
});
```
- Launch Storybook

**Comment by David-Rickard** [comment]
This addon fixes it: https://storybook.js.org/addons/storybook-addon-next
